public/src/client/topic/postTools.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+var utils = require('../../utils');
```

Comment:
This is not necessary and breaks in my browser with `nodebb.min.js?v=jke600bhapg:14759 Uncaught Error: Module name "../../utils" has not been loaded yet for context: _. Use require([])`

`utils` is a global so this can be removed.

Reply:
Ah, okay. I've not used JS before, so I didn't know how to tell what's global and what's not, and the IDE I was using didn't seem to see it unless I `require`d it.

Reply:
So do I change this and then submit a new PR?

Reply:
No, pushing additional commits will reflect the change in the PR 🙂